docs/commands/heap.md

```diff
@@ -79,6 +79,15 @@ gef➤ heap chunks --min-size 16 --max-size 32
 
 ![heap-chunks-size-filter](https://i.imgur.com/AWuCvFK.png)
 
+If heap chunks command gives too many chunks, we can use `--count` argument to limit the number
```

Comment:
this should probably move, because below you talk about the min and max size range being inclusive.

Reply:
oops, missed the line.